client/src/research/BubbleChart.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {percentRightPerProfile} from './calculations';
+import _ from 'lodash';
+import {
+  isSwipe
+} from './functions.js';
+
+class BubbleChart extends Component{
+  grouping(interactions, key){
+    var groups = _.groupBy(interactions, row => {
+      return row.interaction.turn[key];
+    });//lodash group by {Name1:{maker:percent},Name3:{captain:percent}} {maker:{Name1: percent, Name2:percent}}
```

Comment:
This is a helpful comment to see the shape of the data, but please rephrase to indicate that this is what the method is returning, and move this up to above the method name like this:

```
// this is a comment about what the method does
group(interactions, key) {
  ...
}
```